inclusive-design/wecount.inclusivedesign.ca
# fix: improved styling and display of content images from wordpress (resolves #137)

## Description
This PR improves styling and display of content images from WordPress.


## Steps to test

1. Go to deploy preview.
2. Navigate to Views page.
3. Observe the alignment of images in test posts 89-100.
4. Observe the alignment of media in [Journeys through the Digital Innovation Appendix of the Master Innovation and Development Plan](https://deploy-preview-321--wecount.netlify.app/views/journeys-through-the-digital-innovation-appendix-of-the-master-innovation-and-development-plan/) post and [Project We Count evaluates Toronto’s Quayside Master Innovation and Development Plan](https://deploy-preview-321--wecount.netlify.app/views/project-we-count-evaluates-torontos-quayside-master-innovation-and-development-plan/) post.


**Expected behavior:** 

Images should be sized and aligned according to how they were set in WordPress.
## Additional information
N/A



## Related issues
resolves #137
